hs_composite_resource/models.py

```diff
@@ -32,11 +32,18 @@ def set_default_logical_file(self):
         of the resource that is not already associated with a logical file. """
 
         for res_file in self.files.all():
+            # TODO: This following if block will not be needed once we migrate
+            # raster file type to HStore
```

Comment:
@pkdash OK. I think I got it. ```logical_file_new``` is your new HStore experiment which would be renamed as```logical_file``` once this is the route we decide to go with.